### PR TITLE
[T] Bump version

### PR DESCRIPTION
**Pull request by SinisaG**
Bumping version for #187 . 

@viniciuskneves : Do you want to include #200 ? I'll release tomorrow.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "homeday-blocks",
-  "version": "2.5.0",
+  "version": "3.0.0",
```
Comment (SinisaG) [inline package.json]: Is a major version bump, as we have breaking changes (HdInfoBox).

**Comment by viniciuskneves** [comment]
I've just merged #200 🚀

**Comment by viniciuskneves** [comment]
@SinisaG just opened this one and would appreciate to have it in the same bump https://github.com/homeday-de/homeday-blocks/pull/202

**Comment by ilyasmez** [comment]
@SinisaG 
We already have a [release](https://github.com/homeday-de/homeday-blocks/releases/tag/3.0.0) with the version `3.0.0`, but it was released from develop branch 😕 
We can either delete it and release the new one immediately, or we bump to `4.0.0`.

I vote for the first option.

**Comment by viniciuskneves** [comment]
I vote for first opinion hehehe

**Comment by SinisaG** [comment]
> @SinisaG
> We already have a [release](https://github.com/homeday-de/homeday-blocks/releases/tag/3.0.0) with the version `3.0.0`, but it was released from develop branch 
> We can either delete it and release the new one immediately, or we bump to `4.0.0`.
> 
> I vote for the first option, but we have to merge the change logs.

Oh, you're right (https://github.com/homeday-de/homeday-blocks/blob/master/package.json#L3) - that also explains the conflicts in `package.json`. The first option is not so easy though, as you can't overwrite the published version on npm. `npm publish -f` has been deprecated - https://blog.npmjs.org/post/77758351673/no-more-npm-publish-f

We could `unpublish`, wait for 24 hours and then `publish` again. 

Taking this in consideration, I would opt for bumping to `4.0.0`, resolving conflicts and target master. We're dealing with problem/no problem, where out version numbers are getting pretty high. Is a consequence of lot's of changes / stabilizing it. But I think that's fine, as long as we follow semversoning correctly. 

What do you think @homeday-de/frontend-engineering ?

**Comment by ilyasmez** [comment]
@SinisaG 

> We could additionally still delete version 3.0 from git versions and re-release it targeting master - as this is the state of master currently (just to be consistent :) ). Then release version 4.0.0 on top.

I prefer this solution :)

File: package.json [hunk 2]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "homeday-blocks",
-  "version": "3.0.0",
+  "version": "4.0.0",
```
Comment (SinisaG) [inline package.json]: Should be correct now. :+1:

Comment (viniciuskneves) [reply package.json]: Hey @SinisaG you can use [`npm version`](https://docs.npmjs.com/cli/version) in this case. It automatically bumps `package.json` and `package-lock.json` for example. You can check an example here: https://github.com/VueFloripa/vuefloripa.github.io/blob/development/package.json#L8

With this merge we got a mismatch between both files.

**Pull request merged**